packages/radix-vue/src/Combobox/ComboboxRoot.vue

```diff
@@ -226,6 +227,13 @@ function scrollSelectedValueIntoView() {
     selectedElement.value.scrollIntoView({ block: 'nearest' })
 }
 
+function focusOnSelectedElement() {
+  // Find the highlighted element and focus
+  // This helps the screen readers to read the selected value
+  if (selectedElement.value instanceof Element && selectedElement.value.focus)
+    selectedElement.value.focus()
+}
+
```

Comment:
We don't need this focus function as [`aria-activedescendant`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-activedescendant) already handle this

Reply:
I thought the same - but that doesn't work with all screen readers apparently.

Reply:
Icic.. If that's the case then no prob 😁